chapter1.md

```diff
@@ -61,23 +61,25 @@ Model fitting, log-likelihood, and AICc calculations were performed using the ma
 
 # Results
 
-Across all data sets, the negative binomial and Poisson lognormal distributions had very similar average log-likelihoods (within 0.01 of one another). The log-likelihoods for each of these distributions averaged 0.8 units higher than for the log-series distribution and 5 units higher than for the Zipf distribution (corresponding to likelihoods that were twice as high and 140 times as high, respectively).
+Figure 1 shows three example sites with the empirical distribution and associated models fit to the data along with the log-likelihoods of the models. Zipf distributions tend to predict the most rare species followed by the log-series, the negative-binomial, and Poisson log-normal.
```

Comment:
I've moved the Figure 1 up to the end of the models section, but I'm open to moving it up further if folks thing that's better. My thinking was that at this point it helps provide some context for the different models that have just been discussed.